docker/test_env/ethereum_env.go

```diff
@@ -11,6 +11,8 @@ import (
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/google/uuid"
+	"github.com/pelletier/go-toml/v2"
+	"github.com/pkg/errors"
```

Comment:
I thought we got rid of usages of this since it is no longer maintained. We had replaced most usages with fmt.Errorf

Reply:
For reference we had removed all references in this PR back in November: https://github.com/smartcontractkit/chainlink-testing-framework/pull/762

Reply:
There is an ongoing conversation, we haven't found an alternative yet, so I'm okay to keep it.